DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -80,7 +80,7 @@ object BlobSinker extends SinkOperatorFactory {
         hdfsPath = outputPath,
         content = content,
         timeout = Duration.create(ConfigManager.getActiveDictionary().getOrElse(JobArgument.ConfName_BlobWriterTimeout, "10 seconds")),
-        retries = 0,
+        retries = 10,
```

Comment:
Curious to understand why we chose the number of retries as 10 and not something else. Did we see any failures in the test while writing events to blobs ?

Reply:
yes, we can see some timeout error with the blob sinker and the program will exit if there's no retry